city_scrapers/spiders/cook_pharmaceutical_disposal.py

```diff
@@ -0,0 +1,114 @@
+from datetime import datetime, time
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class CookPharmaceuticalDisposalSpider(CityScrapersSpider):
+    name = "cook_pharmaceutical_disposal"
+    agency = "Cook County Pharmaceutical Disposal Advisory Committee"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.cookcountysheriff.org/rx/advisory-committee/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        self.base_url = "https://www.cookcountysheriff.org"
+
+        # Link to current meeting shown as a single image
+        # In case you want to proceed with image OCR
+        # image_url = base_url +
+        #    response.xpath("//div[@class='col-sm-12 ']/h1/img/@src").get()
+        # image_response = requests.get(image_url)
+        # print(image_to_string(Image.open(BytesIO(image_response.content)),lang='eng'))
+
+        # Scraping past meetings
+        for item in response.xpath("//div[@class='col-sm-12 ']/p"):
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Cook County Safe Disposal of Pharmaceuticals Advisory Committee"
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return ADVISORY_COMMITTEE
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        # Dates are not correct for some meetings as they
```

Comment:
To be safe here, we should ignore any meeting where we can't pull the month, date, and year

Reply:
The only issue is that we only can get specific dates from one meeting using the agenda and minutes file names.

Reply:
It looks like we should be able to get it from at least December and July of 2019, although they aren't super clean, which is fine with me if that works for you!